lesson2/hometask_frameq.rb

```diff
@@ -0,0 +1,24 @@
+module M
+  def self.included(base)
+    base.extend(M2)
+    puts base.methods - Object.methods
+  end
+
+  def object_instance
+
+  end
+
+  module M2
+    def self.extended(base)
+      puts base.new.methods - Object.methods
+    end
+    def class_instance
```

Comment:
Add empty line. Definitions of your methods should be separated by one empty line.